.github/workflows/actionlint.yml

```diff
@@ -15,12 +15,13 @@ jobs:
   actionlint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Set up Go 1.x
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v5
         with:
           go-version-file: './go.mod'
           check-latest: true
+          cache: false
```

Comment:
Prior to v4, `cache: false` was the default; we need to preserve `cache: false` to avoid errors in `Go Mod Cache` in go.yml's launcher and package-builder build steps.

I've set `cache: false` everywhere for consistency with the behavior we previously expected.